lib/bits_service_client/logging_http_client.rb

```diff
@@ -37,9 +40,22 @@ def do_request(request, vcap_request_id)
 
       request.add_field('X-VCAP-REQUEST-ID', vcap_request_id)
 
-      @http_client.request(request).tap do |response|
-        @logger.info('Response', { code: response.code, vcap_request_id: vcap_request_id })
+      begin
+       response = @http_client.request(request)
+       @logger.info('Response', { code: response.code, vcap_request_id: vcap_request_id })
+       rescue Net::ReadTimeout => ex
```

Comment:
Something with the indentation got mixed up here. Maybe tabs vs spaces. Not sure. Please check and fix.

Reply:
this should now be fixed with rubocop run 
https://github.com/cloudfoundry-incubator/bits-service-client/pull/3/commits/1679b0027d848b859810b23b9ee94eaa8d5d0e27